content/data-types/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Data Types"
+date: 2023-01-17T16:28:22-06:00
+draft: false
+weight: 2
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Learning Objectives
+
+After completing everything in this chapter, you should be able to do the following:
+
+1. Explain the difference between static and dynamic typing
+1. Write comments in C#
+1. Name the numeric types in C#
+1. Declare variables in C#
+1. Distinguish between value and reference types
+1. Parse data types
+1. Follow proper C# naming conventions
+
+## Key Terminology
+
+Throughout this lesson, you will see lots of new vocabulary. Here is everything that you should make note of:
+
+1. dynamically typed
+1. statically typed
+1. type system
+1. primitive type
+1. class
+1. boxing
+1. unboxing
+1. using 
+1. namespace
+1. assembly
+1. `Console.WriteLine`
+1. `Console.ReadLine`
+1. comments
+1. `char`
+1. array
+1. object
+1. method
+1. static
+1. public
+
```

Comment:
Please reformat based on last weeks conversation.

Reply:
Done!

Reply:
There are two styles with regards to the formatting for Key Terminology.

Style 1 -> Data Types, Classes and Unit Testing
Style 2 -> Control Flow & Collections and Classes Part 2

Suggest on deciding on one style to make the book feel more cohesive.

Reply:
We will do that! Thanks! Merging for now